src/index.html

```diff
@@ -7,6 +7,40 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <main>
+      <div class="calendar">
+        <div class="days"></div>
```

Comment:
```suggestion
        <div class="day"></div>
```
it's a div for one! day. Same as next